lighthouse-core/config/experimental-config.js

```diff
@@ -14,14 +14,28 @@
 const config = {
   extends: 'lighthouse:default',
   audits: [
-    'full-page-screenshot',
+    'byte-efficiency/duplicated-javascript',
+    'legacy-javascript',
   ],
   passes: [{
     passName: 'defaultPass',
     gatherers: [
       'full-page-screenshot',
     ],
   }],
+  audits: [
+    'valid-source-maps',
+  ],
+  // @ts-ignore: `title` is required in CategoryJson. setting to the same value as the default
+  // config is awkward - easier to omit the property here. Will defer to default config.
+  categories: {
+    'best-practices': {
+      auditRefs: [
+        {id: 'duplicated-javascript', weight: 0, group: 'load-opportunities'},
+        {id: 'legacy-javascript', weight: 0, group: 'diagnostics'},
```

Comment:
same–delete these items, they are in default now.

Does `valid-source-maps` show up using this config? I think it needs an entry here.

Reply:
I put `valid-source-maps` in the default-config instead of the experimental-config (which is now unchanged). do we want it in experimental-config?